packages/react/script/components-json/build.ts

```diff
@@ -48,8 +54,12 @@ const components = docsFiles.map(docsFilepath => {
   // Example: src/components/Box/Box.docs.json -> src/components/Box/Box.stories.tsx
   const defaultStoryFilepath = docsFilepath.replace(/\.docs\.json$/, '.stories.tsx')
 
+  const isComponentV2 = docs.importPath === '@primer/react/next'
+  const docsName = String(docs.name).toLowerCase()
+  const componentName = isComponentV2 ? `${docsName}v2` : docsName
```

Comment:
Not sure if this is true for all v2 components 😢 

[packages/react/src/TooltipV2/](https://github.com/primer/react/tree/main/packages/react/src/TooltipV2/)
[packages/react/src/drafts/SelectPanel2](https://github.com/primer/react/tree/main/packages/react/src/drafts/SelectPanel2)
[packages/react/src/Dialog/](https://github.com/primer/react/tree/main/packages/react/src/Dialog/)